modules/ROOT/pages/prerequisites/prerequisites.adoc

```diff
@@ -389,3 +389,19 @@ Ceph Notes::
 
 * Note that you cannot access the same files in CephFS and Ceph-S3. Ceph allows exposing commodity hardware as either blockstorage (RBD), S3 or CephFS. It is not possible to write a file via S3 and then read it via CephFS.
 ====
+
+== Compatible Clients
+
+The mobile clients and the desktop client work just like with ownCloud server when connected to Infinite Scale if used in their current versions:
+
+* Desktop app version 2.11
+* iOS app version 11.10
+* Android app version 2.21
+
+At the moment, only the new ownCloud Web interface offers spaces. These will work with the upcoming client versions:
```

Comment:
```suggestion
The mobile clients and the desktop client work just like with ownCloud server when connected to Infinite Scale if used in their current versions. This means that users can access and synchronize their home folder and recieved shares, but not spaces:

* Desktop app version 2.11
* iOS app version 11.10
* Android app version 2.21

At the moment, only the new ownCloud Web interface offers spaces. Spaces will be available in the following upcoming client versions:
```

Reply:
I precised the text what the impact of the limitations are so readers get a better impression about.

Reply:
"recieved" => "received"